python/servo/package_commands.py

```diff
@@ -674,6 +677,22 @@ def upload_to_s3(platform, package, timestamp):
                 }
             )
 
+            if github_release_id:
+                from github import Github
+                g = Github(os.environ['NIGHTLY_REPO_TOKEN'])
+                nightly_repo = g.get_repo(os.environ['NIGHTLY_REPO'])
+                release = nightly_repo.get_release(int(github_release_id))
+                if '2020' in platform:
+                    asset_name = f'servo-latest-layout2020.{extension}'
+                else:
+                    asset_name = f'servo-latest.{extension}'
+                sha_name = f'{asset_name}.sha256'
```

Comment:
I'd avoid the temporary here, just so that the code below is easier to read.

Reply:
Done in 61fb795